### ISSUE TITLE
WE Elixir of Shadows does not go on CD.

### ISSUE THREAD

**Issue by FrogenYozurtZA** [WarEmu/WarBugs#17473]
If I reactivate EoS very quickly after the ability activates (within around 200ms) the ability goes, I am given the buffs associated with the ability (MS and invis) according to buffhead and the ability goes on CD but as soon as the second activation occurs the CD is removed from my hotbar and addons are also unable to track its CD. Further I am able to reactivate the ability repeatedly and the animation goes off but the ability will not go on CD nor will I receive any of the buffs associated with it. 

After the 60s CD is over I can reactivate the ability as per usual, and can recreate the bug if I reactivate it very shorty after the initial activation.

My intuition is that it is latency related as I have friends with around 50ms who do not experience said bug. I play from South Africa and have 150ms ping to EU. With server latency I would imagine my latency is closer to 200ms.

I can play around this by ensuring I only ever issue a single activation command at a time but this is very hard in the heat of battle especially when having higher latency means it is all the more important to activate abilities as soon as the GCD allows, for reasonable rotation speeds. 

If you need anymore information please don't hesitate to reach out. 

**Thank you from myself and the whole community for devoting your spare time to maintaining this wonderful game for us all!**

Cheers.

**Comment by FrogenYozurtZA** [comment]
Edit: After some testing I have not experienced this with longer CD abilities on any other class such as DoKs Soul Shielding or BOs Challenge.

It seems to only affect EoS on WE afaik.

**Comment by Kaelang** [comment]
Does sound like a latency issue as opposed to a game issue if you are experiencing this where others are not. May be worth looking on the Forums for suggestions to reduce ping.

**Comment by FrogenYozurtZA** [comment]
Unfortunately there is no way to reduce ping as I am in South Africa. While
it may, in part, be due to latency the, fact that no other abilites behave
like this suggests a fault with the implementaion of said abilities
mechanics. That being said I understand the dev team being as small as it
is this is most likely not a priority.

Are there any players from NA that perhaps have experienced similar issues?

**Comment by Kaelang** [comment]
I can keep this open for review, but if it's an individual case I'm leaning towards it being a ping issue.

**Comment by Krimalicious** [comment]
It happens actually to all Elixirs and Fented Position also. I play from brazil latency close to 200ms also.

**Comment by FrogenYozurtZA** [comment]
Yes after joing a WE guild and chatting with them I noticed alot of people
seem to experience this issue. It might well be latency associated, never
the less its only effects these few abilites in WE and not any other
abilities on any other class in my experienced.
